tests/check_framework/urls/contains_tuple.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import include, url
+
+urlpatterns = [
+    url('^', include([
```

Comment:
Just wondering if there's a reason for putting the tuple inside an `include()` instead of in the top level `urlpatterns`?